landing_page/Gemfile

```diff
@@ -50,6 +53,8 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'pry'
```

Comment:
would recommend moving the pry gems to test, development as it is sometimes useful to breakpoint during a test

Reply:
That's a good catch, thanks for spotting it!